create_grids/incident/sps/install_maraston11_pickles.py

```diff
@@ -0,0 +1,154 @@
+"""
+Download Maraston2011 and convert to HDF5 synthesizer grid.
+"""
+
+#Allow the file to use incident_utils 
+
+import numpy as np
+import os
+import argparse
+from pathlib import Path
+import tarfile
+from synthesizer.utils import flam_to_fnu
+from synthesizer.sed import calculate_Q
+from datetime import date
+import wget
+import utils
+import sys
+
+#Allow the file to use incident_utils 
+sys.path.append(os.path.join(os.path.dirname(__file__), '..')) 
+from incident_utils import write_data_h5py, write_attribute, add_log10Q#, __tag__
+
+c = 3*10**10
+
+
+def download_data(input_dir):
+    filename = wget.download(original_data_url)  # download the original data to the working directory
+
+    Path(input_dir).mkdir(parents=True, exist_ok=True)
+
+    # --- untar main directory
+    tar = tarfile.open(filename)
+    tar.extractall(path=input_dir)
+    tar.close()
+    os.remove(filename)
+
+
+
+def make_grid(model, imf, extension):
+    """Main function to convert Maraston 2011 and
+    produce grids used by synthesizer"""
+
+    #synthesizer_model_name = get_model_filename(model)
+    #print(synthesizer_model_name)
+
+    # Define output
+    fname = f"{synthesizer_data_dir}/input_files/{model_name}/{model_name}{extension}_{imf}.hdf5" #ideally find a way to create the grids/model_name folder if it doesn't already exist
```

Comment:
We (try to) abide by the [PEP 8 style guide](https://peps.python.org/pep-0008/), one consequence of which is that there is a maximum line length. To implement this automatically there is a code formatter called [black](https://github.com/psf/black) that you can run on your code, and it will automatically do all the formatting changes for you. It could be worth running this on your script.